### ISSUE TITLE
Support "Interlingua" locale

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons#12178]
A new locale `ia` has been committed via Pontoon, it stands for "Interlingua": https://en.wikipedia.org/wiki/Interlingua.

For QA: "Interlingua" should be listed in the language chooser.
